db/seeds/site_settings.yml

```diff
@@ -313,3 +313,17 @@
   category: AdvancedSettings
   description: >
     The minimum score a question must have to qualify for selection for the Hot Posts sidebar and for being selected as random advertisement.
+
+- name: TableOfContentsThreshold
+  value: 5
+  value_type: integer
+  category: Display
+  description: >
+    The number of child posts before the TOC is shown. If the threashold is not met, do not show TOC at all.
```

Comment:
I'll defer language to @cellio on this one, but please fix the spelling for "threashold" :)

Reply:
The minimum number of child posts before the table of contents is shown.  If the number of child posts is less than this threshold, the TOC is not shown.

(Expand TOC on first use, unless there's already context for that. Rest is just wordsmithing.)